server.js

```diff
@@ -17,6 +17,21 @@ app.use(express.static(__dirname));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
+//Cross-Origin Resource Sharing CORS
+//To enable access of resources from our server
+app.use((req, res, next) => {
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header(
+    'Access-Control-Allow-Headers',
+    'Origin, X-Requested-With, Content-Type, Accept'
+  );
+  if (req.method === 'OPTIONS') {
+    res.header('Access-Control-Allow-Methods', 'PUT, POST, PATCH, DELETE, GET');
+    return res.status(200).json({});
+  }
+  next();
+});
+
```

Comment:
Delete this and use the `cors` npm module (https://www.npmjs.com/package/cors)

Reply:
also try to configure the node server to only use `cors` during development. check `process.env.NODE_ENV === 'production';`